src/init.cpp

```diff
@@ -752,6 +752,20 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
     if (nConnectTimeout <= 0)
         nConnectTimeout = DEFAULT_CONNECT_TIMEOUT;
 
+    // Set the maxlimitertxfee tweak
```

Comment:
This if block isn't needed.  CTweaks are automatically loaded with the information from mapArgs

Reply:
took this out.